### ISSUE TITLE
ttfonts.map not in zhmetrics

### ISSUE THREAD

**Issue by u-fischer** [MiKTeX/miktex#298]
- [x] I have read the [MiKTeX Contributing Guidelines](https://github.com/MiKTeX/miktex/blob/master/CONTRIBUTING.md)

If one (or some driver) tries to use ttfonts.map and this is not found, the on-the-fly installation tries to install zhmetrics. But zhmetrics doesn't contain ttfonts.map. 

See also https://tex.stackexchange.com/questions/487857/iguanatex-ttfonts-map-missing

I had a ttfonts.map, but don't know which package did install it, for the test I renamed it:

~~~
\documentclass{book}
\pdfmapfile{=ttfonts.map}
\begin{document}
ccc
\end{document}
~~~

**Comment by edocevoli** [comment]
After installing the package: `ttfonts.map` is installed here:

```
C:\Program Files\MiKTeX 2.9\fonts\map\ttf2pk\
```

This is the source:

http://mirrors.ctan.org/fonts/psfonts/zhmetrics/fonts/map/ttf2pk/ttfonts.map

**Comment by u-fischer** [comment]
Yes, that where my version is too. I don't know why the OP on tex.sx seems to be missing the map-file (and why it is used at all), but why does a missing map triggers the installation of zhmetrics which doesn't contain the map?

**Comment by QxxxZ** [comment]
Hi,u-fischer(I assume you are the same person on SE),I asked it on LatexSE and lost the password.So I am posting my reply here. 

I don't have MikTex installed on that directory.According to MikTex Log
LinkTargetDirectory:C\User\Username\Appdata\Local\Programs\MikTex 2.9\miktex\bin\×64
Root0:C\User\Username\Appdata\Roaming\MikTex\2.9
Root1:C\User\Username\Appdata\Local\Miktex\2.9
Root2:C\User\Username\Appdata\Local\Programs\MikTex\2.9
Root3:C\ProgramData\Miktrx 2.9
Root4:C\ProgramFiles\Miktex 2.9
UserInstall C\User\Username\Appdata\Local\Programs\Miktex 2.9

I could not understand your first comment on SE also there is nothing in Root4.The console .exe runs from Appdata\Local.

**Comment by u-fischer** [comment]
If you have a single user miktex installation look for the map in `C:\User\Username\Appdata\Local\Programs\MikTex 2.9\fonts\map\ttf2pk`. Beside this, this here is not the right place for a discussion. If you lost your access to tex.sx create a new one and then flag a moderator so that they merge the accounts. Then you can edit your original question.

**Comment by edocevoli** [comment]
It seems that TL does not install this file (for a reason?), so perhaps I should remove it from the `zhmetrics` package.

See https://tug.org/pipermail/tex-live/2011-April/028899.html
